ui/pages/AnvilWorkspace/components/LoadWorkspaceDataForm.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Header, Segment } from 'semantic-ui-react'
+
+import { FILE_FIELD_NAME, PROJECT_DESC_FIELD, GENOME_VERSION_FIELD, FAMILY_FIELD_ID, INDIVIDUAL_FIELD_ID } from 'shared/utils/constants'
+import { BaseBulkContent, BASE_UPLOAD_FORMATS } from 'pages/Project/components/edit-families-and-individuals/BulkEditForm'
+import { INDIVIDUAL_CORE_EXPORT_DATA, INDIVIDUAL_ID_EXPORT_DATA } from 'pages/Project/constants'
+import ReduxFormWrapper from 'shared/components/form/ReduxFormWrapper'
+import { BooleanCheckbox } from 'shared/components/form/Inputs'
+
+const FIELD_DESCRIPTIONS = {
+  [FAMILY_FIELD_ID]: 'Family ID',
+  [INDIVIDUAL_FIELD_ID]: 'Individual ID (needs to match the VCF ids)',
+}
+const REQUIRED_FIELDS = INDIVIDUAL_ID_EXPORT_DATA.map(config => (
+  { ...config, description: FIELD_DESCRIPTIONS[config.field] }))
+
+const BLANK_EXPORT = {
+  filename: 'individuals_template',
+  rawData: [],
+  headers: [...INDIVIDUAL_ID_EXPORT_DATA, ...INDIVIDUAL_CORE_EXPORT_DATA].map(config => config.header),
+  processRow: val => val,
+}
+
+const UploadPedigreeField = React.memo(() =>
+  <Segment>
+    <BaseBulkContent
+      blankExportConfig={BLANK_EXPORT}
+      requiredFields={REQUIRED_FIELDS}
+      optionalFields={INDIVIDUAL_CORE_EXPORT_DATA}
+      uploadFormats={BASE_UPLOAD_FORMATS}
+      actionDescription="load individual data from an AnVIL workspace to a new seqr project"
+      url="/api/upload_temp_file"
+    />
+  </Segment>,
+)
+
+const UPLOAD_PEDIGREE_FIELD = {
+  name: FILE_FIELD_NAME,
+  label: 'Upload Pedigree Data',
+  component: UploadPedigreeField,
+}
```

Comment:
The `label` is not displayed. I don't know why yet.

Reply:
Oh hmm yeah it looks like you do nned to explicitly add the label to the component the way you originally were instead of passing it in as a prop. But you will want to add it in a `<label>` tag, not as a custom styled component. If that throws a warning in the js you can disable it they way we do here: https://github.com/broadinstitute/seqr/blob/master/ui/shared/components/form/Inputs.jsx#L300